bindings/python/src/alert.cpp

```diff
@@ -1029,6 +1057,27 @@ void bind_alert()
         .add_property("dropped_alerts", &get_dropped_alerts)
         ;
 
+    class_<dht_live_nodes_alert, bases<alert>, noncopyable>(
+       "dht_live_nodes_alert", no_init)
+        .add_property("node_id", &dht_live_nodes_alert::node_id)
+        .add_property("num_nodes", &dht_live_nodes_alert::num_nodes)
+        .add_property("nodes", &dht_live_nodes_nodes)
+        ;
+
+    std::vector<sha1_hash> (dht_sample_infohashes_alert::*samples)() const = &dht_sample_infohashes_alert::samples;
+
+    class_<dht_sample_infohashes_alert, bases<alert>, noncopyable>(
+       "dht_sample_infohashes_alert", no_init)
+        //.add_property("endpoint", &dht_sample_infohashes_alert::endpoint)
+        .add_property("endpoint", make_getter(&dht_sample_infohashes_alert::endpoint, by_value())) // TODO: should be correct like this, but in Python endpoint contains only garbage
```

Comment:
this is supposed to invoke the `endpoint_to_tuple` converter, right? when you say "garbage", do you at least get a tuple?

Reply:
I ported the patch to libtorrent 1.2.5 and now it works flawlessly. I guess some other bugfix fixed this as well.